tests/it.xsemantics.dsl.tests/test-gen/fj_lambda_test/it/xsemantics/test/fj/lambda/FjTestsForLambdas.java

```diff
@@ -17,14 +27,7 @@
 import org.eclipse.xtext.xbase.lib.ListExtensions;
 
 /**
- * Copyright (c) 2013-2017 Lorenzo Bettini.
- * All rights reserved. This program and the accompanying materials
- * are made available under the terms of the Eclipse Public License v1.0
- * which accompanies this distribution, and is available at
- * http://www.eclipse.org/legal/epl-v10.html
- * 
- * Contributors:
- *   Lorenzo Bettini - Initial contribution and API
+ * Uses rule invocations as boolean expressions
```

Comment:
This file did not contain an `@author` tag before, just the name. In this case it is only a test expectation.

In general, we did not remove the `@author` tags. To double check this, I just did a
```
git diff upstream/master | grep "@author"
```
and nothing was found.